_posts/patterns/2015-12-11-tables.md

```diff
@@ -0,0 +1,257 @@
+      <td>Son of Arathorn</td>
+    </tr>
+    <tr>
+      <td>Bilbo</td>
+      <td>Baggins</td>
+      <td>Bag End</td>
+    </tr>
+    <tr>
+      <td>Meriadoc</td>
+      <td>Brandybuck</td>
+      <td>Buckland</td>
+    </tr>
+    <tr>
+      <td>Treebeard</td>
+      <td>Fangorn</td>
+      <td>Fangorn Forest</td>
+    </tr>
+  </table>
+</div>
+
+<h3 class="hd-6 example-set-hd">Simplified and compact</h3>
+<div class="example-set">
+  <table class="table table-simplified table-cozy">
+    <tr>
+      <th>First</th>
+      <th>Last</th>
+      <th>Lineage</th>
+    </tr>
+    <tr>
+      <td>Aragorn II</td>
+      <td>Elessar</td>
+      <td>Son of Arathorn</td>
+    </tr>
+    <tr>
+      <td>Bilbo</td>
+      <td>Baggins</td>
+      <td>Bag End</td>
+    </tr>
+    <tr>
+      <td>Meriadoc</td>
+      <td>Brandybuck</td>
+      <td>Buckland</td>
+    </tr>
+    <tr>
+      <td>Treebeard</td>
+      <td>Fangorn</td>
+      <td>Fangorn Forest</td>
+    </tr>
+  </table>
+</div>
+
+<h3 class="hd-6 example-set-hd">Full Semantic Set</h3>
+<div class="example-set">
+  <div class="wrapper-table-scrollable">
+    <table class="table">
+      <caption>Some fantastical towns and their key characteristics</caption>
+      <colgroup class="colgroup">
+        <col class="col col1">
+      </colgroup>
+      <colgroup class="colgroup">
+        <col class="col col2">
+        <col class="col col3 is-highlighted">
+        <col class="col col4">
+        <col class="col col5">
+      </colgroup>
+      <colgroup class="colgroup is-highlighted">
+        <col class="col col6">
+      </colgroup>
+      <thead>
+        <tr>
+          <th>Town Name</th>
+          <th>Founding feature</th>
+          <th>Weaknesses</th>
+          <th>Strengths</th>
+          <th>Primary Inhabitants</th>
+          <th>No. Inhabitants</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr>
+          <th>Pernrith</th>
+          <td>Ancient water source</td>
+          <td>Low economy</td>
+          <td>Rare animal training and rare herbalism</td>
+          <td>People</td>
+          <td class="align-right">15</td>
+        </tr>
+        <tr>
+          <th>Fandralore</th>
+          <td>Strong metal ores</td>
+          <td>Armorsmithing</td>
+          <td>Sustainable hunting and prosperous trade</td>
+          <td>Elves</td>
+          <td class="align-right">20</td>
+        </tr>
+        <tr class="highlight">
+          <th>Kasmord</th>
+          <td>Precious gems</td>
+          <td>Medicine</td>
+          <td>Strong magicians and refined baking</td>
+          <td>Elves</td>
+          <td class="align-right">25</td>
+        </tr>
+      </tbody>
+      <tfoot>
+        <tr class="align-right">
+          <th colspan="5">Total World Inhabitants</th>
+          <td class="align-right">60</td>
+        </tr>
+      </tfoot>
+    </table>
+  </div>
+</div>
+
+
+<h3 class="hd-6 example-set-hd">Responsive (with Column hiding)</h3>
```

Comment:
@frrrances, might be worth adding those descriptions we chatted about in the PR specifically to this example to explain what responsive options/mechanics are available and what our philosophy is on responsive tabular data.

Here's the general markup:

```html
<div class="example-set-description copy-meta">
    <p>[example set up, description, or details]</p>
</div>
```